internal/pkg/earpointer/analysis_test.go

```diff
@@ -891,8 +891,9 @@ func TestVariadicCall(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	want := "{f.a,f.t4,g.t1}: [], {f.b}: [], {f.t0,f.t3,g.ks}: " +
-		"[0->g.t0, 1->f.t2], {f.t1,g.t0}: --> g.t1, {f.t2}: --> f.b"
+	want := "{f.a,f.t4,g.t1}: [], {f.b}: [], " +
```

Comment:
This was not really the suggestion. Can we make a testing utility that takes a non-string type to do the comparison?  Like a map[string]string where the key is the "{f.a,f.t4,g.t1}" and value is the "[]"? That seems like it would simplify maintenance a lot.

Reply:
`cmp.Diff` maybe?

Reply:
Now the non-trivial strings are replaced by a map from the key to the field string. Yes it is much more readable.